packages/typescriptlang-org/src/templates/play.tsx

```diff
@@ -111,7 +111,7 @@ const Play: React.FC<Props> = (props) => {
           div.style.webkitAnimation = ""
         })
 
-        document.getElementById("loading-message")!.innerHTML = `This version of TypeScript <em>(${tsVersion?.replace(/</g, "-")})</em><br/>has not been prepared for the Playground<br/><br/>Try <a href='/play?ts=${latestRelease}${document.location.hash}'>${latestRelease}</a> or <a href="/play?ts=next${document.location.hash}">Nightly</a>`
+        document.getElementById("loading-message")!.innerHTML = `This version of TypeScript <em>(${tsVersion?.replace(/</g, "-")})</em><br/>has not been prepared for the Playground<br/><br/>Try <a href='/play?ts=${latestRelease}${document.location.hash}'>${latestRelease}</a> or <a href="/play?ts=next${document.location.hash}">next</a>`
```

Comment:
Although the tag is "next", we pretty much always refer to it as the nightly build. I would suggest not changing this.

Reply:
Okay. Should `ts=Nightly` instead be the default? I'm unclear why the two names in the first place.

Reply:
For the URLs, I would always use `next` as that's the tag on npm, it's just the human language stuff would be better staying as "nightly".